overrides/home.html

```diff
@@ -0,0 +1,158 @@
+  rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+  sell copies of the Software, and to permit persons to whom the Software is
+  furnished to do so, subject to the following conditions:
+
+  The above copyright notice and this permission notice shall be included in
+  all copies or substantial portions of the Software.
+
+  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+  FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+  IN THE SOFTWARE.
+-->
+
+{% extends "main.html" %}
+
+<!-- Render hero under tabs -->
+{% block tabs %}
+  {{ super() }}
+
+  <!-- Additional styles for landing page -->
+  <style>
+
+    /* Application header should be static for the landing page */
+    .md-header {
+      position: initial;
+    }
+
+    .tx-container {
+      text-align: center;
+      padding-bottom: 5rem;
+      padding-top: 1.8rem;
+      background-color: var(--md-primary-fg-color);
+      background-image: url(assets/hero_background.svg);
+      background-repeat: no-repeat;
+      background-position: bottom;
+      background-size: 101%;
+    }
+
+    @media (prefers-color-scheme: dark) {
+      .tx-container {
+        background-image: url(assets/hero_background_dark.svg);
+      }
+    }
+
+    .tx-hero {
+      padding: .5rem;
+    }
+
+    .tx-hero p {
+      max-width: 800px;
+      margin: 1rem auto;
+    }
+
+    .tx-hero h1, .tx-hero p {
+      color: var(--md-primary-bg-color);
+    }
+
+    
+
+    .tx-hero .md-button {
+      margin-top: .5rem;
+      margin-right: .5rem;
+      color: var(--md-primary-bg-color);
+    }
+    .tx-hero .md-button--primary {
+      color: var(--md-primary-fg-color);
+      background-color: var(--md-primary-bg-color);
+      border-color: var(--md-primary-bg-color);
+    }
+
+    /* Hide table of contents */
+    @media screen and (min-width: 60em) {
+      .md-sidebar--secondary {
+        display: none;
+      }
+    }
+
+    /* Hide navigation */
+    @media screen and (min-width: 76.25em) {
+      .md-sidebar--primary {
+        display: none;
+      }
+    }
+
+    .md-content {
+      margin: -1.5rem auto 0 auto;
+      text-align: center;
+    }
+
+
+    .tabbed-set {
+      background: var(--md-primary-bg-color);
+      box-shadow: 0 0 20px 0px #00000035;
+    }
+
+    .tabbed-set label {
+      color: var(--md-primary-bg-color)
+    }
+
+    .md-content hr {
+      margin: 2.2em auto;
+      max-width: 400px;
+    }
+
+    .md-content article h1, .md-content .md-content__button, .md-content .headerlink, hr:last-of-type, .md-source-date {
+      display: none;
```

Comment:
The langing page is pretty hacky, as you can see a lot of features had to be disabled by simply not rendering them. Thats not nice, not clean, but works. 😋